app/src/components/combobox/ComboBox.tsx

```diff
@@ -46,10 +45,18 @@ export function ComboBox<T extends object>({
   children,
   container,
   size = "M",
+  width,
   ...props
 }: ComboBoxProps<T>) {
   return (
-    <AriaComboBox {...props} css={comboBoxCSS(props.width)} data-size={size}>
+    <AriaComboBox
+      {...props}
+      css={comboBoxCSS}
+      data-size={size}
+      style={{
+        width,
+      }}
+    >
       <Label>{label}</Label>
```

Comment:
This isn't rendering for some reason 
<img width="709" alt="image" src="https://github.com/user-attachments/assets/2ffa6869-b00f-4c78-b46c-a594bc7ee698">

Reply:
oops that's cause i removed teh prop for some reason